pypgstac/setup.py

```diff
@@ -6,15 +6,8 @@
 
 package_data = \
 {'': ['*'],
- 'pypgstac': ['migrations/pgstac.0.2.4-0.2.7.sql',
-              'migrations/pgstac.0.2.4-0.2.7.sql',
-              'migrations/pgstac.0.2.4-0.2.7.sql',
-              'migrations/pgstac.0.2.5-0.2.7.sql',
-              'migrations/pgstac.0.2.5-0.2.7.sql',
-              'migrations/pgstac.0.2.5-0.2.7.sql',
-              'migrations/pgstac.0.2.7.sql',
-              'migrations/pgstac.0.2.7.sql',
-              'migrations/pgstac.0.2.7.sql']}
+ 'pypgstac': ['migrations/pgstac.0.2.5-0.2.8.sql',
+              'migrations/pgstac.0.2.8.sql']}
```

Comment:
I believe the setup.py is generated through poetry, but I manually updated this; removed the duplicate entries which didn't seem necessary.